src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -368,12 +368,26 @@ const MatrixHeader = ({ context, showProjects, project }, reactContext) => {
 
     const rowCount = matrix => matrix.y['biosample_ontology.classification'].buckets.reduce((currentTotalCount, termName) => currentTotalCount + termName['biosample_ontology.term_name'].buckets.length, 0);
 
+    const query = new QueryString(context.search_base);
+    let organism = '';
+    if (query.getKeyValues('replicates.library.biosample.donor.organism.scientific_name').includes('Mus musculus')) {
```

Comment:
I assume the organisms will never be more than human and mouse. If true, can you use conditional operator (: ?)? Also, organism too could be the const.